README.md

```diff
@@ -5,6 +5,11 @@ This repo is designed as a resource for developers building applications that us
 Feel free to create an issue if you have a question, suggestion, or would like to report an issue to the Azure Cache team. We welcome feedback on the cache product itself, the contents of this repo, or anything else. 
 
 ### Table of contents: 
+
+- [Quickstarts](https://github.com/Azure-Samples/azure-cache-redis-samples)
+  - Small projects that demonstrate the essential steps to incorporate Azure Cache for Redis into an app.
```

Comment:
Let's put these links in a new section called "Other Resources" after the "Table of contents" section.

Reply:
Ooh, yeah. That makes sense. Let me move them.